topics/transcriptomics/tutorials/ref-based/tutorial.md

```diff
@@ -1269,6 +1269,8 @@ correction for the variability due to the 2nd factor. In our current case, treat
 > {: .solution}
 {: .question}
 
+<!-- some more explanation of p-values would be nice - I think this is a really confusing topic, more so than PCA... -->
+
```

Comment:
Maybe a small section or tip box on p-values would be nice here?

Reply:
I added a tip box here.

Reply:
This is great, thanks!